### ISSUE TITLE
set soap12 namespace and assign it to a binding

### ISSUE THREAD

**Issue by mjrk** [arskom/spyne#454]
For the soap12 stub to work with the WSDL, there are two things requrired:

* specify the xmlns:soap12="http://schemas.xmlsoap.org/wsdl/soap12/" namespace
* assign this namespace to a binding

Currently there are just constants in `spyne.const.xml`

    NS_WSDL11_SOAP = 'http://schemas.xmlsoap.org/wsdl/soap/'

and `spyne.const.xml_ns`

    soap = 'http://schemas.xmlsoap.org/wsdl/soap/'

I did a quick try-out by replacing them https://github.com/mjrk/spyne/tree/soap12-namespace. This basically makes the WSDL work with SoapUI and Soap 1.2. (and not 1.1. anymore), however, not sure how you plan to make it configurable though.

**Comment by plq** [comment]
we should have `NS_WSDL11_SOAP11` and `NS_WSDL11_SOAP12` and use it properly in Wsdl11 class.

**Comment by mjrk** [comment]
Yes, that's what I thought. Unfortunately I won't have time until September, however, I opened the issue to track the status. Not much missing for Soap 1.2. Thanks for your great work!

**Comment by AllToTheX** [comment]
I created a possible solution to the namespace selection in: AllToTheX/spyne@fd4c3e1
I'm waiting with the pull request for #523 to be merged so the commits stay clean.

If you have another suggestion for selecting the namespace please let me know, i'm happy to implement it.
